akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -630,6 +641,17 @@ complete the current value is emitted downstream.
 
 **completes** when upstream completes
 
+reduce
+^^^^^^
+Start with first element and then apply the current and next value to the given function, when upstream
+complete the current value is emitted downstream. Similar to ``fold``.
+
+**emits** when upstream completes
```

Comment:
it emits aggregated state when upstream just completed. `stream.impl.fusing.Reduce` has only one push and it's pushing aggregate state.

Or your comment was not about emits section?